### ISSUE TITLE
Calloutview with Mapbox 

### ISSUE THREAD

**Issue by RD1991** [nfarina/calloutview#97]
Hi Nick Farina,
Thank you for this lovely project. I need to ask if we can use this CalloutView with Mapbox ?

**Comment by friedbunny** [comment]
SMCalloutView is built in to the various Mapbox SDKs and customizability varies. With the new iOS SDK there's [an open ticket](https://github.com/mapbox/mapbox-gl-native/issues/2211) and an as-yet unreviewed [pull request](https://github.com/mapbox/mapbox-gl-native/pull/3322) that would add the ability to specify your own view — SMCalloutView or some other.

As it stands, it's only necessary to include the SMCalloutView headers in your project. This won't get you very far because your callout view won't be synced with map view, though.

**Comment by RD1991** [comment]
Hi but I can see that "only those have write access can have pull request" ! 
Thank you for your response !

**Comment by friedbunny** [comment]
Yup, if you want to integrate other's/your own code into our project, you'll need to [clone it locally](https://help.github.com/articles/cloning-a-repository/), merge the commits yourself, and then [build the SDK](https://github.com/mapbox/mapbox-gl-native/blob/master/docs/BUILD_IOS_OSX.md).

**Comment by RD1991** [comment]
![screen shot 2015-12-28 at 4 35 54 pm](https://cloud.githubusercontent.com/assets/16399313/12017757/cb8cbd5c-ad80-11e5-88de-ceaa90cdec3b.png)

I want to achieve this ! Is this possible ?

**Comment by friedbunny** [comment]
Customized views like that should be possible in the upcoming v3.1.0 release.

**Comment by RD1991** [comment]
Thanks for the response!
